syft/pkg/license_set_test.go

```diff
@@ -5,6 +5,8 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
+	"github.com/anchore/syft/internal"
+	"github.com/anchore/syft/syft/license"
 	"github.com/anchore/syft/syft/source"
 )
 
```

Comment:
need a case under test for concluded type (or a difference in type) as well as how that interplays with url/location/value/expression.